documentation/api/parameters.md

```diff
@@ -130,7 +130,8 @@ Content-Type: application/x-ndjson
 
 - The target application must use ASP.NET Core.
 - The target application cannot have [Hot Reload](https://learn.microsoft.com/visualstudio/debugger/hot-reload) enabled.
-- `dotnet-monitor` must be set to `Listen` mode, and the target application must start suspended. See [diagnostic port configuration](../configuration/diagnostic-port-configuration.md) for information on how to do this.
+- `dotnet-monitor` must be set to `Listen` mode. See [diagnostic port configuration](../configuration/diagnostic-port-configuration.md) for information on how to do this.
+- If the target application has the `dotnet-monitor` startup hook manually configured then the feature will only work if the target application is started suspended.
```

Comment:
suggestion:

Let's combine these with the below and talk in terms of what .NET version the target app is using
> - If a target application is using .NET 7 then the `dotnet-monitor` startup hook must be configured. This is automatically done in .NET 8+.

Perhaps something like

> If the target application is using .NET 7 then the `dotnet-monitor` startup hook must be manually configured and the target application must start suspended.